sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/crosslanguage/SnowflakeReadRegistrar.java

```diff
@@ -56,11 +54,9 @@ public ReadBuilder() {}
 
     @Override
     public PTransform<PBegin, PCollection<byte[]>> buildExternal(ReadConfiguration c) {
-      SnowflakeCredentials credentials = SnowflakeCredentialsFactory.of(c);
-
       SerializableFunction<Void, DataSource> dataSourceSerializableFunction =
           SnowflakeIO.DataSourceProviderFromDataSourceConfiguration.of(
-              SnowflakeIO.DataSourceConfiguration.create(credentials)
```

Comment:
@purbanow Do you know how to test it? Maybe I should add credentials in the other way?

Reply:
i think in the past we had tests for cross-language using java. I think @piotr-szuberski wrote those tests, but we decided to drop them.